src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -134,6 +134,16 @@ struct H5IsBondiData {
   using group = Cce;
 };
 
+struct H5SpecNormalization {
+  using type = bool;
+  static constexpr Options::String help{
+      "true if automatic corrections for SpEC data impurities should be "
+      "applied automatically based on the `VersionHist.ver` data set in the "
```

Comment:
I'd suggest omitting one of the two uses of "automatic(ally)" in this sentence, it'll be easier to read